packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -491,17 +491,22 @@ export function MediaPlaceholder( {
 				<>
 					{ renderDropZone() }
 					<FormFileUpload
-						variant="primary"
-						className={ clsx(
-							'block-editor-media-placeholder__button',
-							'block-editor-media-placeholder__upload-button'
+						render={ ( { openFileDialog } ) => (
+							<Button
+								onClick={ openFileDialog }
+								variant="primary"
+								className={ clsx(
+									'block-editor-media-placeholder__button',
+									'block-editor-media-placeholder__upload-button'
+								) }
+							>
+								{ __( 'Upload' ) }
+							</Button>
```

Comment:
I moved this to an explicit render prop, so the upsizing can be resolved along with the other Button components surrounding it.

The resulting output should remain the same. This is the `render` prop logic in FormFileUpload:

https://github.com/WordPress/gutenberg/blob/68dd5ce54901a2d5f8bf25ba170d603701426ad4/packages/components/src/form-file-upload/index.tsx#L43-L49

An instance of MediaPlaceholder, with other buttons:

<img src="https://github.com/user-attachments/assets/1f264a02-8756-40d8-8d71-cd0df9ca2269" alt="MediaPlaceholder" width="520">